finality-aleph/src/sync/forest/mod.rs

```diff
@@ -359,25 +359,25 @@ where
 
     /// Updates the provided header, returns:
     /// 1. If required is set whether it became a new explicitly required.
-    /// 2. Otherwise whether it's a new descendant of the highest justified.
+    /// 2. Otherwise whether it's a new header above the favourite block.
```

Comment:
The definition of `above` is not clear to me. How about:
```suggestion
    /// 2. Otherwise whether it's a new header above the favourite block (in terms of block number only).
```

Reply:
Good point, reworded to be more explicit.